apps/condo/domains/notification/adapters/hcmAdapter.js

```diff
@@ -49,6 +51,7 @@ class HCMAdapter {
         [APP_RESIDENT_KEY]: null,
         [APP_MASTER_KEY]: null,
     }
+    #configErrors = []
```

Comment:
what is `#` notation means? private field? as soon as I remember we mark em with `_myPrivateField`

Reply:
in one of my previous PRs colleagues introduced this way for private fields. 
https://github.com/open-condo-software/condo/pull/2699#discussion_r1053309658

Reply:
> what is `#` notation means? private field? as soon as I remember we mark em with `_myPrivateField`

Private fields mark with sharp notation and supported natively by language